contracts/VotingAdapter.vy

```diff
@@ -23,6 +23,10 @@ interface IVoting:
         _supports: bool,
         _executesIfDecided_deprecated: bool,
     ): nonpayable
+    def setDelegate(
```

Comment:
Semantically, it may be better to declare delegate methods in the standalone interface, as it's quite a separate aspect. Something like `IVoteDelegation` or something. Then, in the `delegate()` method, it will look like `IVoteDelegation(DELEGATION_CONTRACT_ADDR).resetDelegate()`, which is more descriptive, in my opinion.

Reply:
Agree